src/utils/api.js

```diff
@@ -6,34 +6,47 @@ const newsApi = axios.create({
 
 export const getArticles = (topic) => {
     if(topic){
-        console.log(topic)
         return newsApi.get(`/articles?topic=${topic}`)
         .then((result) => {
-            console.log(result.data)
             return result.data;
         })
     } else {
         return newsApi.get('/articles').then((result) => {
-            console.log(result.data)
             return result.data;
         })
     }
 }
 
 export const getArticle = (article_id) => {
-    console.log(article_id)
     return newsApi.get(`/articles/${article_id}`)
     .then((result)  => {
-        console.log(result.data, '<- result')
+        console.log(result.data)
         return result.data;
     })
 }
 
-export const getTopics = () => {
-    
+export const increaseVote = (article_id) => {
+    return newsApi.patch(`/articles/${article_id}`, 
+    { inc_votes : 1 } )
+    .then((result)  => {
+        return result.data;
+    })
+}
+
+export const decreaseVote = (article_id) => {
+    return newsApi.patch(`/articles/${article_id}`, 
+    { inc_votes : -1 } )
+    .then((result)  => {
+        return result.data;
+    })
+}
```

Comment:
This works great, you may wish to refactor this in the future to just be 1 function which does both, so you can dynamically pass in +1 or -1, not crucial now though.

Reply:
Yes I agree